sdk/python/feast/infra/registry/sql.py

```diff
@@ -1063,6 +1066,80 @@ def create_project_if_not_exists(self, project):
         if new_project:
             self._set_last_updated_metadata(update_datetime, project)
 
+    def search(
```

Comment:
I was under the impression search should be implemented using full text search. Is pagination implemented in this?

Reply:
The feature view data is stored in one single column "feature_view_proto". So we can't use full text search. About pagination, we can do it in the feature registry code to make the feast code cleaner.

Reply:
About Full Text Search, is that expectation set with Workbench team? 
About Pagination, We are planning to switch to Feast Remote Registry. How does that impact the functionality?

Reply:
@wparsley-eg do you know what workbench team's expectation is?

There is no search in the [remote registry](https://github.com/feast-dev/feast/blob/master/sdk/python/feast/infra/registry/remote.py) anyway. So if we were to switch over, we would have to re-implement it.

Reply:
Remote registry leverages, Sql Registry so we will have the functionality. We don't need to reimplement. Just method definition in remote registry should help until we contribute the functionality. Looking for answers to my questions above.

Reply:
Full text search in this context means "feature view and project names".
That is, the names of all feature views and all projects should be searchable using this functionality. That's what's indicated by the UI design that has been given to us.